draft-ietf-acme-acme.md

```diff
@@ -2002,7 +1999,13 @@ propagating across a cluster or firewall rules not being in place.
 
 Clients SHOULD NOT respond to challenges until they believe that the server's
 queries will succeed. If a server's initial validation query fails, the server
-SHOULD retry the query after some time.  While the server is still trying, the
+SHOULD retry the query after some time, in order to account for delay in setting 
+up responses such as DNS records or HTTP resources.  The precise retry schedule
+is up to the server, but server operators should keep in mind the operational
+scenarios that the schedule is trying to accommodate.  For example, a server
+might perform the first few of retries at intervals 5 or 10 seconds to catch
+automated clients, then back off to intervals of a minute or more to deal with
+manual setup.  While the server is still trying, the
```

Comment:
Overall I think this is a bit of a strange addition. Does the protocol really need to make explicit accommodations in a retry schedule for manual clients? the A in ACME makes me think otherwise. 

The feedback on list was "Any guidance on what "some times" means? At least in terms of a time scale, e.g., seconds, minutes, hours". I think it would be sufficient to give an example of 5 or 10 seconds and not elaborate much further.

Reply:
Agreed.  Dialed back to say basically, "no more than every 5-10 seconds"